Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -900,10 +905,10 @@ private bool CheckCellForCollision(IntVec3 cell)
                     {
                         continue;
                     }
-                    if (BlockerRegistry.CheckForCollisionBetweenCallback(this, LastPos, thing.TrueCenter()))
-                    {
-                        return true;
-                    }
+                    //if (BlockerRegistry.CheckForCollisionBetweenCallback(this, LastPos, thing.TrueCenter()))
```

Comment:
This checks if a pawn added by the adjascent list is standing inside a shield even though the projectile itself doesn't cross the shield.  Ideally we wouldn't try to collide with pawns that aren't in the same cell as our projectile, but overly large pawns take up more than one cell.

Reply:
Now it works without it. Pawns, that partially stays out of shield can now be properly hit